Loaded module: cuda/12.4
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: viswacolab (viswacolab-technical-university-of-denmark) to https://api.wandb.ai. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.8
wandb: Run data is saved locally in /zhome/d5/3/214816/viswa/wandb/run-20250422_124314-7u8c6gm5
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run stellar-firefly-124
wandb: ‚≠êÔ∏è View project at https://wandb.ai/viswacolab-technical-university-of-denmark/maddpg_mrr
wandb: üöÄ View run at https://wandb.ai/viswacolab-technical-university-of-denmark/maddpg_mrr/runs/7u8c6gm5
Traceback (most recent call last):
  File "/zhome/d5/3/214816/viswa/rl_mrr_cont_det_ctrl.py", line 573, in <module>
    cl, al, ent_loss, ent_coeff = agent.learn()
  File "/zhome/d5/3/214816/viswa/sac.py", line 183, in learn
    state, action, reward, new_state, done = self.memory.sample_buffer(self.batch_size)
  File "/zhome/d5/3/214816/viswa/sac.py", line 38, in sample_buffer
    states_ = self.new_state_memory[batch]
KeyboardInterrupt
Traceback (most recent call last):
  File "/zhome/d5/3/214816/viswa/rl_mrr_cont_det_ctrl.py", line 573, in <module>
    cl, al, ent_loss, ent_coeff = agent.learn()
  File "/zhome/d5/3/214816/viswa/sac.py", line 183, in learn
    state, action, reward, new_state, done = self.memory.sample_buffer(self.batch_size)
  File "/zhome/d5/3/214816/viswa/sac.py", line 38, in sample_buffer
    states_ = self.new_state_memory[batch]
KeyboardInterrupt
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/zhome/d5/3/214816/viswa/my_env/lib64/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 151, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/zhome/d5/3/214816/viswa/my_env/lib64/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
